docker/lr-splicedbam2gff/Makefile

```diff
@@ -0,0 +1,17 @@
+IMAGE_NAME = lr-splicedbam2gff
+VERSION = 0.0.1
```

Comment:
how about using the version of spliced_bam2gff, which is currently `[v1.3](https://github.com/nanoporetech/spliced_bam2gff/tree/v1.3)`

Reply:
Sure.  Sounds good.